v1/brokers/redis/redis.go

```diff
@@ -313,7 +285,7 @@ func (b *Broker) nextTask(queue string) (result []byte, err error) {
 	conn := b.open()
 	defer conn.Close()
 
-	items, err := redis.ByteSlices(conn.Do("BLPOP", queue, 1))
+	items, err := redis.ByteSlices(conn.Do("BLPOP", queue, 1000))
```

Comment:
Just want to point out this 1000 must be in **seconds** instead of **milliseconds**.

See: https://redis.io/commands/BLPOP

> The timeout argument is interpreted as an integer value specifying the maximum number of seconds to block. A timeout of zero can be used to block indefinitely.

Reply:
It is in seconds I believe.

Reply:
It's milliseconds

Reply:
I stand corrected.